### ISSUE TITLE
How can I use a WAN port as LAN? 

### ISSUE THREAD

**Issue by 3dzug** [Ysurac/openmptcprouter#3654]
I want to connect a NAS to my openmtcprouter. As I do not want to use eth0 (my computer is connected there) I want to use an unused wan port such eth5.
My setup is: eth0 -> PC eth1 -> mtcp connection eth2 -> mtcp connection eth3 to 5 unused. I do not get a connection if I select dhcp on eth5 and set the ip on the NAS.
Thanks

**Comment by Ysurac** [comment]
In Network->Interfaces disable mptcp on the connection and put it in LAN firewall.

**Comment by Ysurac** [comment]
DHCP client or static IP (can also be DHCP serer in this case).
If you want to use it exactly the same way as current lan, you can create a Bridge with both interface in Network->Interfaces, "Device" tab and put it in LAN firewall. It will be the only one with an IP.
If you want more help, post screenshot of what you have currently in Network->Interfaces.

**Comment by 3dzug** [comment]
Yeah so what I want is that eth0 and eth5 act like a bridge (or switch) with 192.168.100.1 on the server side.Similar to what would happen if I plug eth0 into a switch and distribute it there.Am 14.11.2024 um 12:56 schrieb Ycarus (Yannick Chabanois) ***@***.***>:﻿
DHCP client or static IP (can also be DHCP serer in this case).
If you want to use it exactly the same way as current lan, you can create a Bridge with both interface in Network->Interfaces, "Device" tab and put it in LAN firewall. It will be the only one with an IP.
If you want more help, post screenshot of what you have currently in Network->Interfaces.

**Comment by 3dzug** [comment]
Hi @Ysurac I've set up the bridge but there it is still not working.
Bridge shall be now eth4 to eth0.
![Bridge1](https://github.com/user-attachments/assets/3b018866-780e-40be-
![Bridge2](https://github.com/user-attachments/assets/4a5dfc5b-9cf9-4286-ab8a-40d062a49087)
b759-10615737bb1b)

**Comment by 3dzug** [comment]
I understand that but I do not know how I can do that.
![bridge3](https://github.com/user-attachments/assets/e25cb337-df98-4c38-9e8f-3abc1b749522)

**Comment by Ysurac** [comment]
In Device, choose the bridge. But should be more on LAN interface.

**Comment by 3dzug** [comment]
Can you help me with a step by step guide? Within the interface setting I cant see on lan how I can assign the bridge.
